### DISCUSSION TITLE
视频下载增加av1编码支持

### DISCUSSION THREAD

**Discussion by MasterOfStar** [the1812/Bilibili-Evolved#3052] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/21274896/155206068-54e76a1e-01ae-4de0-b222-97c80f77b085.png)
如图，目前仅支持avc和hevc

**Comment by Geeyun-JY3** [comment]
重复：#2941

**Comment by theblueandblackcrow** [comment]
我记得新版本支持了呀......

**Reply by Geeyun-JY3** [reply]
看 [更新日志](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.3-preview) 和给 #2941 打的 label 确实如此。

**Reply by the1812** [reply]
只有预览版有，我还没下放到正式版

**Reply by MasterOfStar** [reply]
> 只有预览版有，我还没下放到正式版

我说为什么[#2941](https://github.com/the1812/Bilibili-Evolved/discussions/2941)是complete😂
那以后HDR 杜比视界之类的下载会安排上吗(

**Reply by the1812** [reply]
HDR那些不是早就支持了

**Reply by Geeyun-JY3** [reply]
学会善用搜索：
- HDR：#1946、https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.11.0
- 杜比视界：#2596、https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.6-preview
